hack/jenkins/save_build.sh

```diff
@@ -92,22 +91,31 @@ sudo /srp-tools/srp provenance action stop --working-dir $WORKSPACE/provenance
 # declare the git source tree for the build.  We refer to this declaration below when adding source inputs.
 sudo /srp-tools/srp provenance declare-source git --verbose --set-key=mainsrc --path=$WORKSPACE --branch=$BRANCH --working-dir $WORKSPACE/provenance
 
+CI_REGISTRY_IMAGE_AKO=$CI_REGISTRY_PATH/ako
 IMAGE_DIGEST=`sudo docker images $CI_REGISTRY_IMAGE_AKO  --digests | grep sha256 | xargs | cut -d " " -f3`
 echo $IMAGE_DIGEST
-sudo /srp-tools/srp provenance add-output package.oci --set-key=ako-image --action-key=ako-bld --name=${CI_REGISTRY_IMAGE_AKO}  --digest=${IMAGE_DIGEST} --working-dir $WORKSPACE/provenance
+sudo /srp-tools/srp provenance add-output package.oci --set-key=ako-image --action-key=ako-build --name=${CI_REGISTRY_IMAGE_AKO}  --digest=${IMAGE_DIGEST} --working-dir $WORKSPACE/provenance
 
+CI_REGISTRY_IMAGE_AKO_OPERATOR=$CI_REGISTRY_PATH/ako-operator
 IMAGE_DIGEST=`sudo docker images $CI_REGISTRY_IMAGE_AKO_OPERATOR  --digests | grep sha256 | xargs | cut -d " " -f3`
 echo $IMAGE_DIGEST
-sudo /srp-tools/srp provenance add-output package.oci --set-key=ako-operator-image --action-key=ako-bld --name=${CI_REGISTRY_IMAGE_AKO_OPERATOR}  --digest=${IMAGE_DIGEST} --working-dir $WORKSPACE/provenance
+sudo /srp-tools/srp provenance add-output package.oci --set-key=ako-operator-image --action-key=ako-build --name=${CI_REGISTRY_IMAGE_AKO_OPERATOR}  --digest=${IMAGE_DIGEST} --working-dir $WORKSPACE/provenance
```

Comment:
Can we please take out ako-operator related changes from SRP? Reason: We do not want to generate ako-operator image as part of helm processing. So we will skip ako-operator step in jenkins job also.  So generating provenance data also will not be useful in that case.

Reply:
As per the discussion will do change later